packages/nimble-components/src/table/components/header/styles.ts

```diff
@@ -26,6 +26,10 @@ export const styles = css`
         cursor: default;
     }
 
+    :host(.right-align) {
+        justify-content: flex-end;
```

Comment:
I considered `flex-direction: row-reverse`, but that would also reverse the order of sort/grouping indicators, which we don't want.

Reply:
Turns out we _do_ want to reverse the sorting/grouping indicators, so now I'm using `flex-direction` instead.